src/Admin/AfterActivationSiteScan.php

```diff
@@ -84,13 +92,17 @@ public function __construct( RESTPreloader $rest_preloader ) {
 	public static function is_needed() {
 		global $pagenow;
 
+		$react = wp_scripts()->query( self::REACT );
+
 		return (
 			is_admin()
 			&&
 			Services::get( 'dependency_support' )->has_support()
 			&&
 			! is_network_admin()
 			&&
+			( $react instanceof _WP_Dependency && version_compare( $react->ver, '18', '>=' ) )
```

Comment:
@westonruter Please note the `AfterActivationSiteScan` service will only work with React 18 and will not show the Site Scan Notice on plugin/themes activation if WP < 6.2.

Reply:
Unless the latest version of the Gutenberg plugin is active, right?